pkg/apis/servicecatalog/checksum/checksum_test.go

```diff
@@ -77,6 +77,8 @@ func TestBrokerSpecChecksum(t *testing.T) {
 				},
 			},
 		},
+		InsecureSkipTLSVerify: true,
+		CABundle:              []byte{13, 24, 35, 46},
```

Comment:
No, nothing specific. Just a test set of bytes that can test that the cheksum is created correctly.